Source/Views/StackViewController.swift

```diff
@@ -69,4 +75,80 @@ final class StackViewController: UIViewController {
+    ///
+    /// - Parameters:
+    ///   - target: The ViewController which should scroll into focus
+    ///   - relativePosition: An optional parameter to specify the alignement of the target. Depending of
+    ///     the stackViews' orientation some relativePosition values have different meaning: 
+    ///     beginning - top/left, end - bottom/right
+    public func scrollToViewController(target targetViewController: UIViewController,
+                                       position relativePosition: RelativePosition = .center, animated: Bool = false) {
+        
+        guard self.childViewControllers.contains(targetViewController) else {
+            return
+        }
+        
+        var targetOffset = CGPoint(x: 0, y: 0)
+        let targetFrame = targetViewController.view.frame
+        
+        if stackView.axis == .vertical {
+            // keep current horizontal offset
+            targetOffset.x = -scrollView.contentInset.left
+            let topOffset = targetFrame.origin.y - scrollView.contentInset.top
+            let bottomOffset = targetFrame.origin.y + targetFrame.height - scrollView.bounds.height
+            let verticalLimit = scrollView.contentSize.height - scrollView.bounds.height
+            
+            switch relativePosition {
+            case .beginning:
+                targetOffset.y = topOffset
+            case .center:
+                targetOffset.y = topOffset - (topOffset - bottomOffset) / 2
+            case .end:
+                targetOffset.y = bottomOffset
+            }
+
+            if targetOffset.y > verticalLimit {
+                targetOffset.y = verticalLimit
+            }
+            if targetOffset.y < scrollView.contentInset.top {
+                targetOffset.y = scrollView.contentInset.top
+            }
+            targetOffset.y = ensureRangeFor(value: targetOffset.y,
+                                            lowerLimit: scrollView.contentInset.top,
+                                            upperLimit: verticalLimit)
+        } else {
+            // keep current vertical offset
+            targetOffset.y = -scrollView.contentInset.top
+            let leftOffset = targetFrame.origin.x - scrollView.contentInset.left
+            let rightOffset = targetFrame.origin.x + targetFrame.width - scrollView.bounds.width
+            let horizontalLimit = scrollView.contentSize.width - scrollView.bounds.width
+            switch relativePosition {
+            case .beginning:
+                targetOffset.x = leftOffset
+            case .center:
+                targetOffset.x = leftOffset - (leftOffset - rightOffset) / 2
+            case .end:
+                targetOffset.x = rightOffset
+            }
+
+            targetOffset.x = ensureRangeFor(value: targetOffset.x,
+                                            lowerLimit: scrollView.contentInset.left,
+                                            upperLimit: horizontalLimit)
+        }
```

Comment:
Had a general look here and also seems to me that some `ensureRangeFor` calls are not needed.